testing/bsg_cache/lock_test/sweep.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+rm -rf test_result.log
+echo "start test" > test_result.log
+run_test() {
+  make clean
+  echo "################" >> test_result.log
+
+  if [ $2 == 2 ]
+    then
+      echo WAY_ON_LOCKED_P = $1  `expr $1 + 1` >> test_result.log
+      make test_lock_multiway.basic.run WAY_ON_LOCKED_P=$1
+    elif [ $2 == 1 ]
+      then
+        echo WAY_ON_LOCKED_P = $1  >> test_result.log
+        make test_lock2.basic.run WAY_ON_LOCKED_P=$1
+      else
+        echo WAY_ON_LOCKED_P = $1  >> test_result.log
+        make test_lock1.basic.run WAY_ON_LOCKED_P=$1
+  fi
+  make summary >> test_result.log
```

Comment:
when I run ./sweep.sh, I see these error messages.
```
grep: out/test_lock2/simv.log: No such file or directory
grep: out/test_lock_multiset/simv.log: No such file or directory
grep: out/test_lock_multiway/simv.log: No such file or directory
make: *** [summary] Error 2
```

Reply:
This is safe to ignore because it collect statistics of one test at a time, but `make summary` searches result of all 4 tests when it is used to grep the data into test_result.log. I will fixed it with a new make target later